turtle.cabal

```diff
@@ -64,7 +64,7 @@ Library
         text                               < 1.3 ,
         time                               < 1.7 ,
         transformers         >= 0.2.0.0 && < 0.6 ,
-        optparse-applicative >= 0.11    && < 0.14,
```

Comment:
This is the source of the build failures in CI.  The root of the problem is that `criterion-1.1.1.0` has an `optparse-applicative < 0.13.0.0` upper bound so the benchmark suite can't build.  There is an issue open on Stackage to fix this:

https://github.com/fpco/stackage/issues/1784

We need to wait on that to be fixed before we can merge this